Jenkinsfile

```diff
@@ -135,7 +135,7 @@ def postsubmit(gitUtils, bazel, utils) {
       bazel.test('//...')
       sh('bin/init.sh')
       sh('bin/codecov.sh | tee codecov.report')
-      sh('bin/toolbox/presubmit/pkg_coverage.sh')
+      sh('bazel run @com_github_istio_test_infra/toolbox/pkg_check || echo "Failed to run pkg_check"')
```

Comment:
I understand the concern that don't want code coverage to block pr, but do you guys think we should at least expose error and exit at post-submit. Otherwise, no one will notice the error at all.